Say/ViewController.swift

```diff
@@ -61,18 +61,23 @@ class ViewController: NSViewController {
     @IBOutlet var voiceComboBox: NSComboBox! = nil
     /// Save panel for "Export" menu
     let voiceSavePanel = NSSavePanel()
-    
     /// Open panel for "Open" menu
     let textOpenPanel = NSOpenPanel()
     
+    @IBOutlet var datePicker: NSDatePicker! = nil
+    @IBOutlet var alarmButton: NSButton!
+    
+    var alarmTime: Date! = nil
+    var timer: Timer! = nil
```

Comment:
`timer` is too general name. Let'ts name it `alermTimer` to distinguish with other future timers.